docs/af-fundamentals-tutorial/modules/deadlines/pages/testing.adoc

```diff
@@ -0,0 +1,37 @@
+= Testing the Deadline
+:navtitle: Test the handler
+
+Testing behaviour related to deadlines is easy by using either the `SagaTestFixture` or the `AggregateTestFixture` these will include configuration to use a `StubDeadlineManager`.
```

Comment:
```suggestion
Testing behaviour related to deadlines is easy by using either the `SagaTestFixture` or the `AggregateTestFixture`. 
These include configuration to use a `StubDeadlineManager` that records scheduling and the passing of time, so the behavior can be checked thoroughly.
```

Reply:
Leaving out the `StubDeadlineManager' to keep it simple.